querybook/webapp/redux/queryReview/types.ts

```diff
@@ -28,6 +28,18 @@ export interface IQueryReviewState {
     errorMyReviews: string | null;
     errorAssignedReviews: string | null;
     activeTab: 'myReviews' | 'assigned';
+
+    // pagination state
+    myReviewsPage: number;
+    myReviewsHasMore: boolean;
+    assignedReviewsPage: number;
+    assignedReviewsHasMore: boolean;
```

Comment:
not sure if this scalable, also we are resetting when tabs are changed anyway, its fine we just have page and hasMore